### DISCUSSION TITLE
有没有适用于win10的版本

### DISCUSSION THREAD

**Discussion by kongshanpai01** [ttttupup/wxhelper#389] [General]
目前版本3.9.2.23-V6，进行注入操作后，可以启动0.0.0.0:19088端口，但在配置文件如下设置时：
[config]
port=19088
HiddenDll=0

，listddls无法搜索到wxhelper，

在浏览器请求http://127.0.0.1:19088/api/?type=0，返回：{"msg":"not support method","result":"ERROR"}
F12显示为 Internal Server Error（500错误）

Apifox发送请求显示请求发生错误：
Agent 错误：无法请求内网地址 http://127.0.0.1:19088/api/?type=0，请选择其他 Agent.

使用Python调用client.py中的接口，显示报错：
InvalidSchema("No connection adapters were found for '%s'" % url)
requests.exceptions.InvalidSchema: No connection adapters were found for '127.0.0.1:19088/api/?type=0'

不太清楚是缺少哪些配置，哪些流程，还是说ddl没法再win10上跑，有没有支持win10的版本

**Comment by FantasyZhou** [comment]
接口地址不对，把?type=0改成文档中对应的地址

[[/api/checkLogin](typora://app/api/checkLogin)](http://127.0.0.1:19088/api/checkLogin)

具体参考doc文件夹下的md文件

**Comment by liudih** [comment]
3.9.5.81版本之后的api地址变了，/api/?type=0 改为了 /api/checkLogin  等
